assets/l10n/app_en.arb

```diff
@@ -483,5 +483,24 @@
   "notifSelfUser": "You",
   "@notifSelfUser": {
     "description": "Display name for the user themself, to show after replying in an Android notification"
+  },
+  "onePersonTyping": "{typist} is typing…",
+  "@onePersonTyping": {
+    "description": "Text to display when there is a user typing.",
```

Comment:
nit:
```suggestion
    "description": "Text to display when there is one user typing.",
```

In the two other cases below, I'd also say "there is a user typing".

Reply:
Do you mean "there is one user typing"? Or "there are two users typing" and etc?

Reply:
Ah sorry, that was unclear: the text below is fine for those other cases. What I mean is, I would say all three of these situations are accurately described with "there is a user typing". So the text for this case should be more specific in order to distinguish it.